tools/debian/copyright

```diff
@@ -0,0 +1,24 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cockpit
+Source: https://github.com/cockpit-project/cockpit
+
+Files: *
+Copyright: 2013-2015  Red Hat, Inc.
```

Comment:
Also other contributors? We could link to the github [cockpit contributors page](https://github.com/cockpit-project/cockpit/graphs/contributors) here.